feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/TimetableScreen.kt

```diff
@@ -141,11 +141,14 @@ fun TimetableScreen(
                 when (uiState.timetable) {
                     is TimetableUiState.Empty -> Text("Empty")
                     is TimetableUiState.GridTimetable -> {
+                        val timetableGridUiState = requireNotNull(uiState.timetable.timetableGridUiState[selectedDay])
                         TimetableGrid(
-                            timetable = requireNotNull(uiState.timetable.timetableGridUiState[selectedDay]).timetable,
+                            timetable = timetableGridUiState.timetable,
                             timeLine = null, // TODO
                             onTimetableItemClick = onTimetableItemClick,
                             selectedDay = selectedDay,
+                            isBookmarked = { id -> timetableGridUiState.timetable.bookmarks.contains(id) },
+                            onBookmarkClick = { id -> onBookmarkClick(id.value) },
```

Comment:
Here, we are using the processing for the “is TimetableUiState.ListTimetable” branch as-is.